README.md

```diff
@@ -53,6 +53,23 @@ docker run -it codechain-io/codechain:branch_or_tag_name
 
 This should result in CodeChain running.
 
+#### Making local database and keys persistent
+
+CodeChain depends on the local database and keys commonly stored under the directories `keys` and `db`. A Docker container is independent of host environment and other Docker images. Therefore, in case of running a new Docker container with an image with a new CodeChain version or even with the same image, the database and keys are not persistent. To solve the problem, one can take advantage of Docker's volume option. With the command below,
```

Comment:
CodeChain depends on the local database and keys commonly stored under the directories `keys` and `db`. A Docker container is independent of host environment and other Docker images. Therefore, when running a new Docker container with an image with a new CodeChain version or even with the same image, the database and keys are not persistent. To solve the problem, one can take advantage of the Docker's volume option. With the command below,

Reply:
Thank you